.github/workflows/main.yml

```diff
@@ -77,19 +77,37 @@ jobs:
         password: ${{ secrets.PYPI_PASSWORD }}
 
   build-manylinux:
-    name: manylinux2010_x86_64 (all versions)
+    name: ${{ matrix.platform }} (all versions)
     runs-on: ubuntu-latest
     needs: [build-src]
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - manylinux2010_x86_64
+          - manylinux2014_aarch64
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+      - name: Set up QEMU
+        if: matrix.platform == 'manylinux2014_aarch64'
+        uses: docker/setup-qemu-action@v1
       - name: Build wheel for manylinux2010_x86_64
+        if: matrix.platform == 'manylinux2010_x86_64'
         uses: RalfG/python-wheels-manylinux-build@v0.3.3-manylinux2010_x86_64
         with:
           python-versions: 'cp36-cp36m cp37-cp37m cp38-cp38 cp39-cp39'
           build-requirements: 'pybison>=0.2.8 six>=1.10.0 watchdog tqdm tornado livereload'
           system-packages: 'atlas-devel flex'
           pre-build-command: 'sh .github/workflows/pre-build-script-manylinux.sh'
+      - name: Build wheel for manylinux2014_aarch64
+        if: matrix.platform == 'manylinux2014_aarch64'
+        uses: RalfG/python-wheels-manylinux-build@v0.3.3-manylinux2014_aarch64
```

Comment:
Can we modify-- uses: RalfG/python-wheels-manylinux-build@v0.3.3-manylinux2014_aarch64
instead of manylinux2014_aarch64, use some env variable ??